homework-3/src/views/PatientProfile.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import BoxRow from '../components/common/Boxes/StylBoxRow';
+import BoxFilter from '../components/Boxes/BoxFilter';
+import BoxRowPadding from '../components/common/Boxes/StylBoxRowPadding';
+import BoxPatients from '../components/common/Boxes/StylBoxPatients';
+import BoxDoctor from '../components/common/Boxes/StylBoxDoctor';
+import BoxFlexColumn from '../components/common/Boxes/StylBoxFlexColumn';
+import BoxPatientContent from '../components/common/Boxes/StylBoxPatientContent';
+import BoxBtnPatients from '../components/common/Boxes/StylBoxBtnPatients';
+import BoxPatientsList from '../components/common/Boxes/StylBoxPatientsList';
+import TitleName from '../components/common/Titles/StylTitleName';
+import TitlePatients from '../components/common/Titles/StylTitlePatients';
+import SubTitlePostion from '../components/common/Titles/StylSubTitlePosition';
+import AvatarActive from '../components/AvatarActive';
+import BtnPatients from './../components/BtnsComponents/BtnPatients';
+import BtnCreateAppointment from './../components/BtnsComponents/BtnCreateAppointment';
+import BtnPatientsHiddenMob from './../components/BtnsComponents/BtnPatientsHiddenMob';
+import CardPatient from '../components/CardPatientProfilePatient/CardProfilePatient';
+import Loader from './../components/Loader/Loader';
+
+import patient from '../img/avatars/avatar-patient.png';
+import avatar2 from '../img/avatars/patients-13.png';
+import logo from '../img/logo.png';
+
+const DoctorProfile = ({ title }) => {
+  React.useEffect(() => {
+    document.title = title || '';
+  }, [title]);
+
+  const [loading, setLoading] = React.useState(true);
+
+  React.useEffect(() => {
+    const loadData = async () => {
+      await new Promise((r) => setTimeout(r, 2000));
+
+      setLoading((loading) => !loading);
+    };
+
+    loadData();
+  }, []);
+
+  if (loading) {
+    return <Loader />;
+  } else {
+    return (
+      <BoxPatients>
```

Comment:
a lot of boilerplate code 
check [solution ](https://github.com/Lemout17/itrex-hw-3/blob/hw-3/src/components/mainDoctor/patientsList/patientsList.js) for better understanding